app/data/clustering.py

```diff
@@ -0,0 +1,38 @@
+import json
+import os
+
+cutoff = 0
+
+def get_smiles_json(smiles):
+  nodes = []
+  edges = []
+
+  for smile_name, smile_data in smiles.items():
+    smile_edges = []
+
+    smile_node = {}
+    smile_node['murcko'] = smile_data['murcko']
+    smile_node['data'] = {}
+    smile_node['data'] = {'id': smile_name }
+    nodes.append(smile_node)
+    for sim, similarity_coefficient in smile_data['similarities'].items():
+
+      # TODO: use actual similarity cutoff
+      if similarity_coefficient >= cutoff and similarity_coefficient != 1:
```

Comment:
Should we verify first that two different smiles can have the same murcko? Like we discussed on Sunday?

Reply:
Yeah good catch- I forgot about this! If it's possible that two have the same murcko, then this will have to change and we may also have to store the similarities differently (since they're currently keyed by the murcko value). I might merge soon, and if we get an answer in the meeting then we can log an issue to refactor it accordingly.